### PR TITLE
Changes done by other user

### PR DESCRIPTION
**Pull request by KetanBhangale04**
Changes done by other user

### CODE REVIEW

File: app/src/main/java/com/example/NewsSampleApp/dao/NewsDao.kt [hunk 1]
```diff
@@ -8,7 +8,7 @@ import retrofit2.Callback
 import retrofit2.Response
 
 class NewsDao(val listener: ISuccessListener) {
-
+//comment added by KetanBhangale04 User
```
Comment (KetanBhangale) [inline app/src/main/java/com/example/NewsSampleApp/dao/NewsDao.kt:11]: ok

File: app/src/main/java/com/example/NewsSampleApp/MainActivity.kt [hunk 2]
```diff
@@ -28,6 +28,7 @@ class MainActivity : AppCompatActivity(), INewsAdapter, ISuccessListener {
         //comment added
         //comment added
         //comment added
+        //comment added by KetanBhangale04 User
```
Comment (KetanBhangale) [inline app/src/main/java/com/example/NewsSampleApp/MainActivity.kt:31]: is this required

Comment (KetanBhangale04) [reply app/src/main/java/com/example/NewsSampleApp/MainActivity.kt:31]: Yes for documentation.

Comment (KetanBhangale) [reply app/src/main/java/com/example/NewsSampleApp/MainActivity.kt:31]: ok

**Pull request merged**